content/theme/hugo-eureka.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Eureka"
+github: https://github.com/wangchucheng/hugo-eureka
+demo: https://themes.gohugo.io/theme/hugo-eureka/
+author: C. Wang
+date: 2020-11-22 # Enter the date you submitted the theme YYYY-MM-DD
+github_branch: master # Specify the repos default branch
+ssg:
+  - Hugo
+cms:
+  - No CMS
+css:
+  - Tailwindcss
+archetype:
+  - Posts
```

Comment:
This isn't the specific Hugo archetypes, its more the type of website. You can use `Documentation` and `Blog` here